features/login.feature

```diff
@@ -1,7 +1,7 @@
 Feature: Check login and logout
 
     Scenario: Login with the valid credential
-      Given I am an analyst AND I am logged into OSIM
+      Given I am an analyst AND want to log into OSIM
       Then I am able to log into OSIM
```

Comment:
If you want to use your branch and create a PR, I did the change because on the pipeline this extra login was almost always failing and because we have the condition on the script, all other tests where skipped